README.md

```diff
@@ -85,19 +87,17 @@ First login to your balena account via `balena login`, and then create a new
 fleet:
 
 ```shell
-$ balena fleet create --type fincm3 DebugApp
-Fleet created: unicorn/debugapp (fincm3, id 1853079)
+$ balena fleet create --type raspberrypi4-64 DebugApp
+Fleet created: slug "ryanh/debugapp2", device type "raspberrypi4-64"
```

Comment:
AFAIK the fleet create command requires that you specify an org. How did it get `ryanh/debugapp2` when you asked to create `DebugApp`?

Reply:
Asking the right questions ⬆️ 
Including one more suggestion to keep fleet, device names as generic as possible. And, phasing out the word app from the docs. So I would suggest using balenaFleet or DebugFleet instead.

Reply:
@klutchell It doesn't require an org, it will default to the logged in account's default AFAIK. It defaulted to that slug `ryanh/debugapp2` because I already had a `ryanh/debugapp`. I can update this and re-run the sequence too to make sure I copy it out exactly!

Reply:
@vipulgupta2048 While I like your suggestion of using fleet instead of app in the fleet name, I don't think it's valuable at this time to go entirely through the entire thing again, just to change the name. I wish I had thought to do this first. Perhaps it would be best to circle back on a future update (when cli/api/supervisor/engine/os etc have been further updated and I'll need to update log output and commands anyway). What do you think?

Reply:
I figured out a way to do it more easily @vipulgupta2048 , I'll update the requested sections so far from you and Kyle and push another commit